addons/l10n_bf/data/template/account.tax.group-bf.csv

```diff
@@ -0,0 +1,4 @@
+"id","name","name@fr","tax_payable_account_id","tax_receivable_account_id"
+"tax_group_0","VAT 0%","T.V.A. 0%","pcg_4431","pcg_4452"
```

Comment:
We tend not to do so, I think (and the main point is to have accounts)

Reply:
I agree but then on the invoice, we get the recap with some stuff in 0% while it shouldn't be in the 0%.
0% is suppose to be stuff subject to vat that was taxed at 0% while exempted means it can't be taxed.
I don't remember what's our policy about that.

Reply:
Hello @Levizar , as far as I know we don't have policy about this topic. It is very country specific and depends on local regulations. I believe there is no specific requirements in Burkina Faso regarding the distinction on the invoice between the exempted operation and the 0% VAT. So I wouldn't change anything.